src/main/java/com/imsweb/mph/MphInput.java

```diff
@@ -29,6 +29,8 @@
 
     private String _txStatus;
 
+    private String _surgeryOfPrimarySite;
```

Comment:
@bekeles is this supposed to be a standard NAACCR field? If that's the case, could you use the same field name as defined by the NAACCR XML standard? 

https://github.com/imsweb/naaccr-xml/blob/master/src/main/resources/naaccr-dictionary-180.xml

It would actually helped if you put the NAACCR Item Number as a comment on top of each these fields so there is no question of whether it's a standard field or not...

Reply:
Aaron added that, I will check.

Reply:
The rule which was using this property was later removed. I just removed the property from this class.